### PR TITLE
Changed miles to kilometers in some areas.

### PR DESCRIPTION
**Pull request by davidwhitman**
Minor changes, per a conversation here: https://groups.google.com/forum/#!category-topic/endless-sky/content-creation/VdhDjPk0ZxI

### CODE REVIEW

File: data/map.txt [hunk 1]
```diff
@@ -18216,7 +18216,7 @@ planet Haze
...
 	description `	Very few regions of this planet remain untouched by human hands, but a few of the particularly mountainous or marshy areas have been set aside as nature preserves. More than half of the indigenous life forms have gone extinct since human settlement began here four centuries ago.`
-	spaceport `The spaceport is constructed purely for efficiency and functionality. It is a long building, stretching in a straight line across more than a mile. Through the center, on tracks elevated at various levels, run monorail trams for carrying passengers and cargo. Along the sides of the building are docking bays alternating with storage rooms. The wide corridors are packed both with people and with robotic freight carts.`
+	spaceport `The spaceport is constructed purely for efficiency and functionality. It is a long building, stretching in a straight line across more than a kilometer. Through the center, on tracks elevated at various levels, run monorail trams for carrying passengers and cargo. Along the sides of the building are docking bays alternating with storage rooms. The wide corridors are packed both with people and with robotic freight carts.`
```
Comment (retrue) [inline data/map.txt]: It should be better to write here "stretching in a straight line across almost two kilometers".   :p

Comment (davidwhitman) [inline data/map.txt]: To keep the conversion? :P I considered that too lol

**Comment by endless-sky** [comment]
Technically speaking, "outside" and "outside of" are equally valid grammar - I actually prefer the former - and "hill region" is a perfectly legitimate use of a noun adjunct. But, I'll merge this anyway.

**Comment by davidwhitman** [comment]
You're right, next time I'll avoid changing things outside the scope of the issue.

**Pull request merged**